client/rspack.config.js

```diff
@@ -18,16 +19,19 @@ const getOutputPath = (app, folder, filename) => {
 
 // Mapping from package name to exposed global variable.
 const exposedDependencies = {
+  // expose-loader may have caveats in Rspack,
+  // see https://github.com/web-infra-dev/rspack/issues/5479
   'focus-trap-react': 'FocusTrapReact',
   'react': 'React',
   'react-dom': 'ReactDOM',
   'react-transition-group/CSSTransitionGroup': 'CSSTransitionGroup',
   'draft-js': 'DraftJS',
```

Comment:
When I first tried Rspack, this was still unsupported (https://github.com/web-infra-dev/rspack/issues/5479). Now it seems to be working correctly, but I don't know if there's anything else we need to consider.